### PR TITLE
review of apiary documentation

### PR DESCRIPTION
**Pull request by andersonluisribeiro**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Apiary documentation update

* **What is the current behavior?** (You can also link to an open issue here)
The apiary documentation is outdated

* **What is the new behavior (if this is a feature change)?**
Apiary documentation updated

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
This PR is connected to dojot/dojot#1051

* **Other information**:

### CODE REVIEW

File: src/docs/apiary.apib [hunk 1]
```diff
@@ -73,6 +93,16 @@ Returns all currently configured flows
                         "label": "Tempalte"
                     }
                 ],
+                "flowRemoteNodes": [
+                    {
+                        "id": "kelvin",
+                        "image": "raulnegreiros/kelvin:3.0.0-alpha2"
```
Comment (raulnegreiros) [inline src/docs/apiary.apib]: hum... it is not nice to put personal things on dojot's repository. It should be nice to change it to dojot

Comment (andersonluisribeiro) [reply src/docs/apiary.apib]: Hi @raulnegreiros, thanks for your feedback. I have sent de corrections.

**Comment by andersonluisribeiro** [comment]
It's necessary to merge this pull request  for the build to be successful: https://github.com/dojot/data-manager/pull/6

File: src/docs/apiary.apib [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
+
+# Import / Export
+Before describing the DataManager APIs, we must explain the import and export functionality. 
+When you export data from Dojot, a JSON file is generated for you. That JSON file contains
+all current templates, devices, flows and remote nodes in your environment. You can save this file
+to import this data later. The import functionality uses this file to regenerate data
+inside Dojot. However, when you import that file, then the templates, devices, flows and remote nodes will
+be created again. It's important to know that this operation is destructive. This means that, 
+if you have templates, devices, flows or remote nodes in your environment, all of them will be
+deleted during the import process, before they are generated. So, be careful.
```
Comment (giovannicuriel) [inline src/docs/apiary.apib]: Maybe point out that this is valid for a particular tenant (and not all of them)?

Comment (andersonluisribeiro) [reply src/docs/apiary.apib]: That sounds nice!

**Pull request merged**